src/app/admin/components/surveillance/edit.html

```diff
@@ -20,7 +20,7 @@ <h4 class="modal-title" id="edit-surveillance-label"><span ng-if="vm.workType ==
     <div class="row" ng-if="vm.workType !== 'initiate'">
       <div class="col-sm-3"><strong>Surveillance initiated by:</strong></div>
       <div class="col-sm-9">
-        <span>{{ vm.surveillance.authority === 'ROLE_ADMIN' ? 'ONC' : 'ONC-ACB' }}</span>
+        <span>{{ vm.surveillance.authority === 'ROLE_ADMIN' || vm.surveillance.authority === 'ROLE_ONC' ? 'ONC' : 'ONC-ACB' }}</span>
```

Comment:
The surveillance authority can never be "ROLE_ADMIN", right? As long as that's the case I think it should be removed from the conditional
```suggestion
        <span>{{ vm.surveillance.authority === 'ROLE_ONC' ? 'ONC' : 'ONC-ACB' }}</span>
```

Reply:
I left it there, because I wasn't sure why it was there to begin with.  I'll remove it.